packages/product-recommendations-quiz-app/product-quiz-ui/pages/quiz/answers/quiz/[id].js

```diff
@@ -0,0 +1,305 @@
+          }
+          removeAnswer={(id) => removeAnswer(id)}
+          question={question}
+        />
+        <Form onSubmit={handleSubmitAnswers}>
+          <Button
+            submit
+            disabled={!question || isSubmitting}
+            primary
+            loading={isSubmitting}
+          >
+            Save Answers
+          </Button>
+        </Form>
+        <Button
+          disabled={!question || isSubmitting}
+          onClick={(event) => {
+            const _id = _.uniqueId();
+            event.preventDefault();
+            addAnswer({
+              _id,
+              text: "",
+              sequence: 1,
+              question: question,
+            });
+          }}
+        >
+          Add answer
+        </Button>
+      </Card>
+    </Layout.Section>
+  );
+};
+
+export default function Answers() {
+  const router = useRouter();
+
+  const quizId = router.query.id;
+  const [quiz, refresh] = useFindOne(api.quiz, quizId, {
+    select: {
+      id: true,
+      title: true,
+      body: true,
+      questions: {
+        edges: {
+          node: {
+            id: true,
+            title: true,
+            body: true,
+            sequence: true,
+            imageUrl: true,
+            answers: {
+              edges: {
+                node: {
+                  id: true,
+                  text: true,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  });
+
+  const currentQuiz = quiz.data;
+  if (currentQuiz) {
+    const currentQuestions = currentQuiz.questions.edges;
+    console.log(currentQuestions);
+    const quizTitle = currentQuiz ? currentQuiz.title : "Untitled";
+
+    currentQuestions.sort((a, b) => a.node.sequence - b.node.sequence);
+    const mappedQuestions = currentQuestions.filter((q) => {
+      q.node.answers.edges.length > 0;
+    });
+
+    return (
+      <Layout>
+        <Layout.Section>
+          <Card
+            title={`Add answers to questions for the ` + quizTitle + ` quiz.`}
+          >
+            {currentQuiz &&
+              currentQuestions.map((q, i) => {
+                return (
+                  <QuestionAnswerForm
+                    key={i}
+                    _id={q.node.id}
+                    question={q.node}
+                  />
+                );
+              })}
+          </Card>
+        </Layout.Section>
+        <Layout.Section>
+          <Card title={`Add results and map quiz`}>
+            <Stack>
+              <></>
+              <Button
+                onClick={() =>
+                  router.push(`/quiz/mapping/quiz/${currentQuiz.id}`)
+                }
+              >
+                On to adding results!
+              </Button>
+            </Stack>
+          </Card>
+        </Layout.Section>
+      </Layout>
+    );
+  } else {
+    return (
+      <Layout>
+        <Layout.Section>
+          <Card title={`There was an issue fetching the current quiz.`}></Card>
```

Comment:
Because you're only passing a static string here, you can just use double-quotes:
```suggestion
          <Card title="There was an issue fetching the current quiz."></Card>
```